hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -44,9 +44,9 @@ class Handle
   }
 
   explicit Handle(const InterfaceDescription & interface_description)
-  : prefix_name_(interface_description.prefix_name),
-    interface_name_(interface_description.interface_info.name),
-    handle_name_(prefix_name_ + "/" + interface_name_)
+  : prefix_name_(interface_description.get_prefix_name()),
+    interface_name_(interface_description.get_interface_name()),
+    handle_name_(interface_description.get_name())
   {
     // As soon as multiple datatypes are used in HANDLE_DATATYPE
     // we need to initialize according the type passed in interface description
```

Comment:
Are we missing the final changes of the variants here?. I don't find the changes in #1689 as well
The changes in the hardware component interfaces would kinda work only with the final version of the Handle changes right?

Reply:
I saw that you are wrapping them with shared_ptrs from the existing objects, this should be fine and would work for backward compatibility

Reply:
sry i did quite catch what you mean.